### PR TITLE
[fuchsia] Use root-ssl-certificates in v2 runner manifests

### PR DESCRIPTION
**Pull request by ypomortsev**
* Add a `use` for the `root-ssl-certificates` to ensure BoringSSL can read the certificates at /config/ssl
* Reformat manifest with `cmc format`

Fixes `HandshakeException`s when establishing a secure connection:

```
HandshakeException: Handshake error in client (OS Error:
        CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(handshake.cc:393))
<asynchronous suspension>
```

Depends on changes that offer this directory to the runners, e.g. https://fxrev.dev/737597

Bug: https://fxbug.dev/111712

### CODE REVIEW

**Comment by ypomortsev** [comment]
@akbiggs PTAL!

**Review by akbiggs — APPROVED** [review]

**Comment by akbiggs** [comment]
LGTM. Repo policy is that we need two Flutter org approvers on each CL that's from a non-Flutter org member, so we'll need @lin-erik's approval before adding the autosubmit label and merging.

**Review by lin-erik — APPROVED** [review]

**Pull request merged**
